nft-market/components/MyNFT/mynft.tsx

```diff
@@ -0,0 +1,78 @@
+import NFT from "@/components/NFT/NFT";
+import { NFTProps } from "@/components/NFT/NFT";
+import { displayAllNFT } from "@/utils/mint";
+import { useState, useEffect } from "react";
+import { WalletConnection } from "calimero-sdk"
+import * as nearAPI from "near-api-js";
+import { config } from "../../utils/calimeroSdk"
+import translations from "../../constants/en.global.json";
+
+interface MyNFTProps {
+    walletConnection: WalletConnection | undefined;
+}
+
+interface MyContract extends nearAPI.Contract {
+    storage_deposit(options: any, gas: number, amount: string): Promise<any>;
+}
+
+export default function MyNFT({ walletConnection}: MyNFTProps) {
+    const [nfts, setNfts] = useState<NFTProps[]>([]);
+
+    const depositStorage = async () => {
+        console.log(walletConnection);
+        
+        if(walletConnection) {
+            const wallet = walletConnection;
+            const contract = new nearAPI.Contract(
+                wallet.account(),
+                config.marketContract,
+                {
+                    changeMethods: ["storage_deposit"],
+                    viewMethods: [],
+                }
+            ) as MyContract;
+            try {
+            const res = await contract["storage_deposit"](
```

Comment:
check how contract calls were written in other examples

Reply:
Refactored them and added all call methods to the utils file callMethods.ts